warehouse/models/mart/gtfs/fct_daily_scheduled_trips.sql

```diff
@@ -75,7 +75,16 @@ fct_daily_scheduled_trips AS (
         stop_times_grouped.trip_last_arrival_sec,
         stop_times_grouped.service_hours,
         stop_times_grouped.contains_warning_duplicate_primary_key AS contains_warning_duplicate_stop_times_primary_key,
-        stop_times_grouped.contains_warning_missing_foreign_key_stop_id
+        stop_times_grouped.contains_warning_missing_foreign_key_stop_id,
+
+        DATE_ADD(service_index.service_date,
+            INTERVAL CAST((TRUNC(SAFE_DIVIDE(stop_times_grouped.trip_first_departure_sec, 86400))) AS INT64) DAY) AS activity_date,
+
+        TIME(TIMESTAMP_SECONDS((stop_times_grouped.trip_first_departure_sec - (CAST((TRUNC(SAFE_DIVIDE(stop_times_grouped.trip_first_departure_sec, 86400))) AS INT64) * 86400))))
```

Comment:
```suggestion
TIME(TIMESTAMP_SECONDS(MOD(trip_first_departure_sec, 86400)))
```
Can do this instead of the formulas for first/last